src/main/java/se/sundsvall/document/api/model/DocumentType.java

```diff
@@ -0,0 +1,73 @@
+package se.sundsvall.document.api.model;
+
+import static io.swagger.v3.oas.annotations.media.Schema.RequiredMode.REQUIRED;
+
+import java.util.Objects;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+
+@Schema(description = "DocumentType model.")
+public class DocumentType {
+
+	@NotBlank
+	@Schema(description = "Id for the document type)", example = "EMPLOYMENT_CERTIFICATE", requiredMode = REQUIRED)
```

Comment:
Use "Identifier" instead of "Id"? Simply since "Id" may be interpreted as having some db pk connection (imo).

Regardless, there's an extra ending parenthesis at the end of the description text.